### DISCUSSION TITLE
how to move data from one step to second step

### DISCUSSION THREAD

**Discussion by kzmKZuzmo** [damianricobelli/stepperize#63] [Q&A] [answered]
Hi, 
I have like 3 steps 

and when I click next in the first step I want to move this data to the second step and in the 3step I want to see step from the first and from the second ist that possible ? is there a way how to passing data from step to step ?

**Comment by damianricobelli** [comment, marked as answer]
https://stepperize.vercel.app/docs/react/api-references/hook#metadata is your answer
